iotdb/src/main/java/org/apache/iotdb/db/query/control/FileReaderManager.java

```diff
@@ -109,46 +119,62 @@ private void clearUnUsedFilesInFixTime() {
   public synchronized TsFileSequenceReader get(String filePath, boolean isUnClosed)
```

Comment:
I think parameter `isUnClosed ` can be replaced with `isClosed `. `isClosed ` is more intuitive.

Reply:
fixed